src/components/screens/consul/Proposals/NewProposal.js

```diff
@@ -149,6 +151,8 @@ export const NewProposal = ({ navigation, data, query }) => {
     setIsLoading(true);
 
     if (newProposalData.image) {
+      if (URL_REGEX.test(newProposalData.image)) return;
```

Comment:
in which cases are there absolute urls for images, that we need to check here?

Reply:
We need this control if it contains a propsal image and the user wants to edit it. The user may want to edit only the text without deleting the picture. If such a situation occurs, we should not send the absolute url value to the server.

Reply:
[e74baff](https://github.com/ikuseiGmbH/smart-village-app-app/pull/539/commits/e74baffa719ec66f2d89cb7518205b6a0c11dd46)